tests/Malarzm/Tests/Collections/ObjectSetTest.php

```diff
@@ -24,6 +24,17 @@ public function testCantAddObjectTwice()
         $this->assertCount(1, $coll);
     }
 
+    public function testClearClearingHashes()
+    {
+        $o = new \stdClass();
+        $coll = new ObjectSet([ $o ]);
+        $this->assertCount(1, $coll);
+        $coll->clear();
+        $this->assertCount(0, $coll);
```

Comment:
This is true, however next `->assertCount(`)` was failing because `->add()` was NOP